types/decimal_test.go

```diff
@@ -0,0 +1,70 @@
+package types_test
+
+import (
+	"github.com/hazelcast/hazelcast-go-client/types"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"testing"
+)
+
+func TestDecimal_String(t *testing.T) {
```

Comment:
Could you convert the tests to the table-driven format with `t.Run`, similar to our other tests?

Reply:
I have converted the tests to table-driven format but should I also convert the test cases? https://github.com/hazelcast/hazelcast-go-client/pull/919/commits/423776327600fa7083cab2e7bc23ed11cd7dd79e